.github/workflows/ci-build.yml

```diff
@@ -3,14 +3,17 @@ name: ci
 on:
   pull_request:
     branches:
-      - master
+      - windows-actions
```

Comment:
Before-merge this should be changed back.

Reply:
Done, thanks!